debugview/build.gradle

```diff
@@ -0,0 +1,29 @@
+apply from: "$rootProject.projectDir/library.gradle"
+
+android {
+    namespace 'com.revenuecat.purchases.debugview'
+
+    defaultConfig {
+        minSdkVersion 21 // Compose requires minSdkVersion 21
+    }
+
+    buildFeatures {
+        compose true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion '1.3.2'
+    }
+}
+
+dependencies {
+    implementation platform('org.jetbrains.kotlin:kotlin-bom:1.8.0')
```

Comment:
Had to update kotlin to 1.8 in this module, it didn't work even with 1.7.20 unless I downgraded compose a lot. Considering this is meant to be a new library, we can require Kotlin 1.8 I think.